packages/venia-ui/lib/queries/getCategoryList.graphql

```diff
@@ -20,4 +20,8 @@ query categoryList($id: Int!) {
             }
         }
     }
+    storeConfig {
+        id
+        category_url_suffix
```

Comment:
We should use the `url_suffix` field off the category query response and not the store config, for now.